### DISCUSSION TITLE
For Single or Multiple Select, Support Circular Selection

### DISCUSSION THREAD

**Discussion by FusionFont** [charmbracelet/huh#556] [Ideas]
Like the **charm** app, when navigating through a list or a set of options, reaching the end of the list will loop back to the beginning, and vice versa.

**Comment by bashbunni** [comment]
Hey @FusionFont, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

I do think this makes sense to add at some point given it's behaviour seen across our projects. That said, the decision will depend on the entire team.

**Reply by banks117** [reply]
Hey all! This was something I came across as well using `huh` for a project. It is a pretty small change so I put up a PR for this.
https://github.com/charmbracelet/huh/pull/755
cc: @bashbunni @FusionFont
